Update UG intro to feature notes (#286)
AY2324S1-CS2103T-F11-1/tp: docs/2101UserGuide.md

To get started, take a look at our [quick start](#quick-start) guide!

This guide uses the following features to make it easier for you to navigate around:
* Words that look like [this](#about) can be clicked to jump to the related section.
* Words that look like `this` refer to commands which you can type into the command box of CCACommander.
* Words that look like <kbd>this</kbd> refer to keyboard keys that you can press.

<div markdown="block" class="alert alert-warning">
:warning: Boxes with the :warning: icon contain important information that you should read.
</div>

<div markdown="block" class="alert alert-info">
:information_source: Boxes with the :information_source: icon contain additional useful information.
</div>

<div markdown="block" class="alert alert-primary">
:bulb: Boxes with the :bulb: icon contain additional tips and tricks to help you get the most out of CCACommander.
</div>

{:toc}

--------------------------------------------------------------------------------------------------------------------

## Quick start
<div markdown="block" class="alert alert-warning">

> :warning: We assume that you have some basic knowledge of terminal commands. You can learn how to do so with [this guide](https://www.freecodecamp.org/news/command-line-for-beginners/)

Reviewer: ```suggestion
:warning: We assume that you have some basic knowledge of terminal commands. You can learn how to use the terminal with [this guide](https://www.freecodecamp.org/news/command-line-for-beginners/).
```

**Result Display**: View messages from CCACommander regarding your commands. <br>
**Member List**: View the members stored in CCACommander. <br>
**Event List**: View the events stored in CCACommander. <br>
**Storage Location**: Locate where your CCACommander data is stored in your computer.

### Command History
You can quickly recall previously entered commands for convenience as CCACommander stores almost all (see the info card below to see what commands are stored) commands entered by the user. <br/>
Upon selecting the command box, you can use the <kbd>↑</kbd> to <kbd>↓</kbd> key to navigate through the commands entered .

Example:
1. You have just marked attendance for the first member in the member list using the editEnrolment m/1 e/1 r/Present command.
2. You can then use <kbd>↑</kbd> to get editEnrolment m/1 e/1 r/Present
3. You can edit the member index to get editEnrolment m/2 e/1 r/Present to mark the second member as present.

<div markdown="block" class="alert alert-info">:information_source: CCACommander will not store 2 of the same commands if they were entered twice in a row. Likewise, "commands" containing exclusively of blank spaces will not be stored as well.

> Upon entering the <kbd>⌫ Backspace</kbd> or the <kbd>↵ Enter</kbd> key, users will have to re-navigate from the most recent command as it was treated as an edit or entry of command.

Reviewer: ```suggestion
Upon entering the <kbd>⌫ Backspace</kbd> or the <kbd>↵ Enter</kbd> key, users will have to re-navigate from the most recent command as it will be treated as an edit or entry of command.
```
